subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -1208,6 +1208,10 @@ static bool send_request(struct hawkbit_context *hb_context, enum hawkbit_http_r
 
 void hawkbit_reboot(void)
 {
+	if (IS_ENABLED(CONFIG_HAWKBIT_REBOOT_NONE)) {
+		LOG_WRN("Reboot disabled, not rebooting");
+		return;
+	}
```

Comment:
put this after the hawkbit_event_raise, also the warning is not needed, as it is expected behavior, as HAWKBIT_REBOOT_NONE has to be enabled by the user

Reply:
Good points, done.